docs/surrealql/datamodel/geometries.mdx

```diff
@@ -81,9 +81,9 @@ A GeoJSON LineString value for storing a geometric path.
 :::
 
 ```surql
```

Comment:
Good catch. Correct example should be this one:

![image](https://github.com/surrealdb/docs.surrealdb.com/assets/6053067/435acbe7-995b-4d95-8561-acf333e307d3)